icaruscode/TPC/Calorimetry/NormalizeTPC_tool.cc

```diff
@@ -128,15 +128,15 @@ icarus::calo::NormalizeTPC::ScaleInfo icarus::calo::NormalizeTPC::GetScaleInfo(u
   }
 
   // Set the cache
-  fScaleInfos[timestamp] = thisscale;
+  fScaleInfos[run] = thisscale;
 
   return thisscale;
 }
 
 double icarus::calo::NormalizeTPC::Normalize(double dQdx, const art::Event &e, 
     const recob::Hit &hit, const geo::Point_t &location, const geo::Vector_t &direction, double t0) {
   // Get the info
-  ScaleInfo i = GetScaleInfo(e.time().timeHigh());
+  ScaleInfo i = GetScaleInfo(e.id().runID().run());
```

Comment:
Skip the middle men!
```suggestion
  ScaleInfo i = GetScaleInfo(e.run());
```